.github/workflows/washboard.yml

```diff
@@ -82,6 +84,7 @@ jobs:
         run: tar -C ./packages/washboard-ui/dist -zcvf washboard.tar.gz .
 
       - name: Upload Artifact
+        if: ${{ matrix.wash-version == '0.29.2' }}
```

Comment:
Is it possible to have this config in a central section? `env` section maybe? I'm thinking this as `WASH_CURRENT_VERSION` and the matrix versions in a `WASH_TEST_VERSIONS`. If they're next to each other, they're easy to update in future

Reply:
Yeah I can definitely do that, will add a `release-version` option

Reply:
So I refactored to have it look like this:

```yaml
      matrix:
        wash:
          - version: 0.27.0
          - version: 0.28.1
          - version: 0.29.2
            release: true
```

That should make it a bit easier to update these things together! Thanks for the suggestion